integration/container/stop_windows_test.go

```diff
@@ -0,0 +1,69 @@
+package container // import "github.com/docker/docker/integration/container"
+
+import (
+	"context"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/integration/internal/container"
+	"github.com/docker/docker/internal/test/request"
+	"gotest.tools/assert"
+	"gotest.tools/poll"
+	"gotest.tools/skip"
+)
+
+// TestStopContainerWithTimeout checks that ContainerStop with
+// a timeout works as documented, i.e. in case of negative timeout
+// waiting is not limited (issue #35311).
+func TestStopContainerWithTimeout(t *testing.T) {
+	skip.If(t, testEnv.OSType == "windows")
```

Comment:
This doesn't look right; this is in a `_windows` file, but skipped?

Reply:
This is being temporarily skipped because the timings are a problem. There are differences in Linux and Windows error codes and handling etc., however, some additional work is required to get the timing right.